### PR TITLE
bugfix/5438,#5463-Add modal window if reason is "Not taken out" in admin order history

### PR DESCRIPTION
**Pull request by NadiiaStoiko**
In admin user story if status "Not taken out" add ability open window with reason, why it not taken out. Added styles.

### CODE REVIEW

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts [hunk 1]
```diff
@@ -16,23 +18,61 @@ export class UbsAdminOrderHistoryComponent implements OnDestroy, OnChanges {
...
 
   openDetails() {
     this.pageOpen = !this.pageOpen;
   }
 
-  showPopup(element: any, popup: any) {
-    if (element.target.offsetWidth < element.target.scrollWidth) {
-      popup.toggle();
-    }
+  showPopup(orderHistoryId) {
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts]: showPopup(orderHistoryId: number): void

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts [hunk 2]
```diff
@@ -16,23 +18,61 @@ export class UbsAdminOrderHistoryComponent implements OnDestroy, OnChanges {
...
+    this.orderHistory.forEach((order) => {
+      if (order.id === orderHistoryId && order.result === ordersStutuses.NotTakenOutUA) {
+        this.openNotTakenOutReason(orderHistoryId);
+      }
+    });
+  }
+
+  openNotTakenOutReason(orderHistoryId) {
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts]: the same

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts [hunk 3]
```diff
@@ -16,23 +18,61 @@ export class UbsAdminOrderHistoryComponent implements OnDestroy, OnChanges {
...
+    console.log('this.orderNotTakenOutReason', this.orderNotTakenOutReason);
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts]: remove console.log

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts [hunk 4]
```diff
@@ -16,23 +18,61 @@ export class UbsAdminOrderHistoryComponent implements OnDestroy, OnChanges {
...
+        id: orderHistoryId,
+        isFromHistory: true,
+        orderID: this.orderInfo.generalOrderInfo.id,
+        description: this.orderNotTakenOutReason.description,
+        images: this.orderNotTakenOutReason.images
+      }
+    });
+  }
+
+  getNotTakenOutReason(orderId: number) {
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts:67]: add return type  - void

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts [hunk 5]
```diff
@@ -16,23 +18,61 @@ export class UbsAdminOrderHistoryComponent implements OnDestroy, OnChanges {
...
+    this.orderService
+      .getNotTakenOutReason(orderId)
+      .pipe(takeUntil(this.destroy$))
+      .subscribe((data: any) => {
+        console.log('data', data);
+        this.orderNotTakenOutReason = data;
+      });
   }
 
   getOrderHistory(orderId: number) {
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts]: the same

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts [hunk 6]
```diff
@@ -41,6 +81,10 @@ export class UbsAdminOrderHistoryComponent implements OnDestroy, OnChanges {
       .pipe(takeUntil(this.destroy$))
       .subscribe((data: IOrderHistory[]) => {
         this.orderHistory = data;
+        console.log(this.orderHistory, 'this.orderHistory');
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts]: remove console.log

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts (hunk 4 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts:67]: added fix

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts (hunk 2 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts]: added fix

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts (hunk 3 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts]: added fix

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts (hunk 4 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts:67]: added fix

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts (hunk 5 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts]: added fix

File: src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts (hunk 6 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-order-history/ubs-admin-order-history.component.ts]: added fix

**Review by KKatiaa — APPROVED** [review]

**Review by atsa21 — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

**Review by ruslan-teh — APPROVED** [review]

**Pull request merged**